main.c

```diff
@@ -28,6 +28,7 @@ static const char usage[] =
 	"                                  meaning as in CSS.\n"
 	"      --border-size <px>          Border size.\n"
 	"      --border-color <color>      Border color.\n"
+	"      --border-radius <px>        Corner radius\n"
```

Comment:
If this is described as pixels, does it really make sense to parse it as a double? It's also used as an int later on which leads me to believe that it doesn't need to be a float.